internal/controller/util/pvcs_util.go

```diff
@@ -29,6 +29,9 @@ const (
 
 	PodVolumePVCClaimIndexName    string = "spec.volumes.persistentVolumeClaim.claimName"
 	VolumeAttachmentToPVIndexName string = "spec.source.persistentVolumeName"
+
+	// Consistency group label
+	ConsistencyGroupLabel = "ramendr.openshift.io/consistency-group"
```

Comment:
The same label is already defined in `volumereplicationgroup_controller.go`. That one has already been refactored in [this PR](https://github.com/RamenDR/ramen/pull/2241). We'll correct it before one or the other is merged first.